fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -3424,58 +3423,58 @@ private ParseNode parseBackupRestoreStatement(ParserRuleContext context) {
             }
 
             if (specifiedFunction) {
-                if (allFunction) {
+                if (allMarker.contains(AbstractBackupStmt.BackupRestoreAllMarker.FUNCTION)) {
```

Comment:
directly `import xxx.AbstractBackupStmt.BackupObjectType;`
Then usage is simpler.

Reply:
fixed